inc/shortcodes/class-facebook.php

```diff
@@ -89,6 +89,7 @@ public static function callback( $attrs, $content = '' ) {
 			'#https?://(www)?\.facebook\.com\/video\.php\?v=[\d]+#',
 			'#https?:\/\/www?\.facebook\.com\/+.*?\/videos\/[\d]+\/#',
 			'#https?://(www)?\.facebook\.com\/permalink\.php\?story_fbid=[\d]+&id=[\d]+#',
+			'#https?:\/\/www?\.facebook\.com\/+.*?\/photos\/a\.[\d]+\.[\d]+\.[\d]+\/([\d])+/#',
```

Comment:
Will it always be `a.[0-9].[0-9].[0-9]`, or should we accommodate an arbitrary number of periods?